src/migrations/mapping_api_version_0_0_5.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs-extra')
+const semver = require('semver')
+const toolbox = require('gluegun/toolbox')
+const yaml = require('js-yaml')
+const { loadManifest } = require('./util/load-manifest')
+const { getGraphTsVersion } = require('./util/versions')
+
+// If any of the manifest apiVersions are 0.0.5, replace them with 0.0.6
+module.exports = {
+  name: 'Bump mapping apiVersion from 0.0.5 to 0.0.6',
+  predicate: async ({ sourceDir, manifestFile }) => {
+    // Obtain the graph-ts version, if possible
+    let graphTsVersion
+    try {
+      graphTsVersion = await getGraphTsVersion(sourceDir)
+    } catch (_) {
+      // If we cannot obtain the version, return a hint that the graph-ts
+      // hasn't been installed yet
+      return 'graph-ts dependency not installed yet'
+    }
+
+    let manifest = loadManifest(manifestFile)
+    return (
+      // Only migrate if the graph-ts version is >= 0.23.0...
+      // Coerce needed because we may be dealing with an alpha version
+      // and in the `semver` library this would not return true on equality.
+      semver.gte(semver.coerce(graphTsVersion), '0.23.0') &&
```

Comment:
Change here to `0.24.0`. Yes, without the `alpha` because there's a coersion here.